src/manufacturer/manufacturer.component.tsx

```diff
@@ -151,6 +161,11 @@ function ManufacturerComponent() {
                       <IconButton
                         size="small"
                         aria-label={`Edit ${item.name} manufacturer`}
+                        onClick={() => {
+                          setEditManufacturerDialogOpen(true);
+                          setSelectedManufacturer(item);
+                          setManufacturer(item);
+                        }}
```

Comment:
Same as https://github.com/ral-facilities/inventory-management-system/pull/97#discussion_r1386439036 could you add test that covers the opening an closing of the edit dialog to cover these lines.